test/omen-subgraph.js

```diff
@@ -846,140 +939,303 @@ describe('Omen subgraph', function() {
+      value: arbitrationCost,
+    });
     const marketBData = {
-      Question: 'Will Ethereum 2.0 Phase 0 launch before 2021?',
-      'Market URL':`https://omen.eth.link/#/${fpmm.address}`
-    }
-    await marketsTCR.addItem(gtcrEncode({ columns, values: marketBData }), { from: creator, value: arbitrationCost})
-    increaseTime(10)
-    await marketsTCR.executeRequest(itemID, { from: creator })
-    const itemIDB = await marketsTCR.itemList(1)
-    await marketsTCR.executeRequest(itemIDB, { from: creator })
-
-    await marketsTCR.removeItem(itemIDB, '', { from: creator, value: arbitrationCost })
-    increaseTime(10)
-    await marketsTCR.executeRequest(itemIDB, { from: creator })
+      Question: "Will Ethereum 2.0 Phase 0 launch before 2021?",
+      "Market URL": `https://omen.eth.link/#/${fpmm.address}`,
+    };
+    await marketsTCR.addItem(gtcrEncode({ columns, values: marketBData }), {
+      from: creator,
+      value: arbitrationCost,
+    });
+    increaseTime(10);
+    await marketsTCR.executeRequest(itemID, { from: creator });
+    const itemIDB = await marketsTCR.itemList(1);
+    await marketsTCR.executeRequest(itemIDB, { from: creator });
+
+    await marketsTCR.removeItem(itemIDB, "", {
+      from: creator,
+      value: arbitrationCost,
+    });
+    increaseTime(10);
+    await marketsTCR.executeRequest(itemIDB, { from: creator });
 
-    await advanceBlock()
+    await advanceBlock();
     await waitForGraphSync();
-    expect((await querySubgraph(`{
+    expect(
+      (
+        await querySubgraph(`{
       fixedProductMarketMaker(id: "${fpmm.address.toLowerCase()}") {
         klerosTCRregistered
         curatedByDxDaoOrKleros
       }
-    }`)).fixedProductMarketMaker).to.deep.equal({ klerosTCRregistered: true, curatedByDxDaoOrKleros: true })
-  })
+    }`)
+      ).fixedProductMarketMaker
+    ).to.deep.equal({
+      klerosTCRregistered: true,
+      curatedByDxDaoOrKleros: true,
+    });
+  });
+
+  step("submit task to gelato", async function () {
+    const zeroAddress = "0x0000000000000000000000000000000000000000";
+
+    const provider = {
+      addr: zeroAddress,
+      module: zeroAddress,
+    };
+
+    const condition = {
+      inst: zeroAddress,
+      data: zeroAddress,
+    };
+
+    const action = {
+      addr: zeroAddress,
+      data: zeroAddress,
+      operation: 0,
+      dataFlow: 0,
+      value: 0,
+      termsOkCheck: false,
+    };
+
+    const task = {
+      conditions: [condition],
+      actions: [action],
+      selfProviderGasLimit: 0,
+      selfProviderGasPriceCeil: 0,
+    };
+
+    const {
+      receipt: { blockHash },
+      logs,
+    } = await gelatoCore.submitTask(provider, task, 0, { from: reporter });
+
+    const submissionArgs = logs.find(
+      ({ event }) => event === "LogTaskSubmitted"
+    ).args;
+    // console.log(submissionArgs);
+    await web3.eth.getBlock(blockHash);
+    // Fails here: graph won't sync
+    await waitForGraphSync();
```

Comment:
The test stalls here. After submitting a gelato task we verify that the `LogTaskSubmitted` indeed appears on the blockchain. However the graph will not sync to the new block height.